tests/pkiunittest.py

```diff
@@ -179,7 +179,7 @@ def user_revoke(self, *opts):
         if not self.reqid and '--help' not in opts:
             raise CertFileError('Could not revoke cert due to missing request ID\n')
         args = opts + ('--test', self.reqid, 'osg-pki-tools unit test - user revoke')
-        cmd = (os.path.join(SCRIPTS_PATH, 'osg-user-cert-revoke'),) + args
+        cmd = ('/bin/sh', os.path.join(SCRIPTS_PATH, 'osg-user-cert-revoke'),) + args
```

Comment:
Just curious, is there a reason the scripts under `SCRIPTS_PATH` are not executable?